[DM-2266]: Added information that software item will be deleted on last version deletion. (#1441)
SoftwareAG/c8y-docs: content/device-management-application/managing-device-data-bundle/managing-software.md

In the details of a specific software, hover over the version entry you want to delete and click the delete icon. The software version will be deleted from the software details.

> If the last version of a software is deleted, the corresponding parent software is deleted as well.

Reviewer: @BeateRixen @rssag We had a discussion while working on https://github.softwareag.com/IOTA/cumulocity-ui/pull/3911 that on the surface there is no hierarchy of software items and we nowhere call anything a "parent software" (the context was the confirmation prompt we initially implemented when deleting the last version).
Can we formulate the sentence by not referring to "parent" software? E.g.:
"If the last version of a software is deleted, the software is entirely removed from the repository."

Author: Yep, I was also wondering if this is a good idea. Will remove it :)

> When the last version in a software is deleted, this will delete its corresponding parent software as well.

Reviewer 2: ```suggestion
If the last version of a software is deleted, the corresponding parent software is deleted as well.
```
